### PR TITLE
Add module crm_lead_comprehensive.

### PR DESCRIPTION
**Pull request by yajo**
Fields needed for MPW.

cc @rafaelbn @dgzurita.

### CODE REVIEW

File: crm_lead_comprehensive/README.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+* If there is any field that exists in the *Company information* area, use it
+  when creating the company.
+* For other information, use the same as what exists in the upper side of the
+  lead.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (pedrobaeza) [inline crm_lead_comprehensive/README.rst:50]: Remove this part

File: crm_lead_comprehensive/README.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+* If you unchecked the option *Use same adress for billing*, use that address
+  information for the company and set it as invoice address.
+* If there is any field that exists in the *Company information* area, use it
+  when creating the company.
+* For other information, use the same as what exists in the upper side of the
+  lead.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
```
Comment (pedrobaeza) [inline crm_lead_comprehensive/README.rst:48]: Remove this link, as this repo doesn't have runbot

**Comment by pedrobaeza** [comment]
Maybe _crm_lead_extra_info_ reflects better the usage of the module.

**Comment by yajo** [comment]
This is being restructured, so closing for now.

**Pull request closed without merging**